iidrees/Events-Manager
# #156979173 Unit and Integration testing

#### What does this PR do?

This PR updates the tests broke down the the test files in order to make them independent of each other i.e. unit tests and integration tests and then tests for models are added.

#### Description of Task to be completed?
N/A
#### How should this be manually tested?

clone the repo or update your copy
in the root directory run `npm test`
the tests should run and the test coverage should show

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#156979173
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
